inst/tinytest/test-cl-content-handler.R

```diff
@@ -1,8 +1,7 @@
 # Test HTTPErrorFactory class
 
-obj = ContentHandlers
-app = Application$new()
-on.exit(obj$reset())
+obj = RestRserve:::ContentHandlersFactory$new()
```

Comment:
we don't necessary need to test `ContentHandlersFactory$new()` since it is part of private API as of now. We need to test `ContentHandlers ` and its side effects.

Reply:
If we skip test `ContentHandlersFactory` coverage will be decreased. We must call class and it methods directly.
Side effects tested in the test-cl-application.R (see last several lines).

Reply:
Ok, then need to add a extensive comment  about that

Reply:
Also I think testing all included classes directly is good practice.
I split tests for the classes and their instances (which used in apps).

Reply:
Interesting discussion https://stackoverflow.com/questions/105007/should-i-test-private-methods-or-only-public-ones

Reply:
In our case some private methods provide a critical functionality. So tests must have in this case.

Reply:
I like idea about decomposition of the complex methods and move it with separate classes or functions with own tests.

Reply:
Added comment/note about coverage to the tests files.